### ISSUE TITLE
--source-directory option broken?

### ISSUE THREAD

**Issue by jessesquires** [realm/jazzy#177]
I'm trying to build docs for [JSQDataSourcesKit](https://github.com/jessesquires/JSQDataSourcesKit).

My project requires the Xcode 6.3 beta, but I switch to the Xcode 6.2 (6C131e) command line tools for jazzy. (Which works fine.)

There's a root workspace with an example and framework project.
````
JSQDataSourcesKit.xcworkspace

|-- Example/
    |-- Example.xcodeproject

|-- JSQDataSourcesKit/
    |-- JSQDataSourcesKit.xcodeproj
````

Here's the way I'm currently (successfully) building docs. I'm specifically running jazzy from the `JSQDataSourcesKit/` framework directory. This [works](http://www.jessesquires.com/JSQDataSourcesKit/).

````bash
cd JSQDataSourcesKit
jazzy -o ../_docs -a 'Jesse Squires' -u 'https://twitter.com/jesse_squires' -m 'JSQDataSourcesKit' -g 'https://github.com/jessesquires/JSQDataSourcesKit'
cd ..
````

When trying to use the `--source-directory` option, jazzy fails.
````bash
jazzy --source-directory './JSQDataSourcesKit/'

Running xcodebuild -dry-run
Could not parse compiler arguments from `xcodebuild` output.
2015-03-13 17:59:34.086 xcodebuild[33284:2218432] [MT] PluginLoading: Required plug-in compatibility UUID A16FF353-8441-459E-A50C-B071F53F51B7 for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/Lin.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2015-03-13 17:59:34.087 xcodebuild[33284:2218432] [MT] PluginLoading: Required plug-in compatibility UUID A16FF353-8441-459E-A50C-B071F53F51B7 for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/KPRunEverywhereXcodePlugin.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2015-03-13 17:59:34.088 xcodebuild[33284:2218432] [MT] PluginLoading: Required plug-in compatibility UUID A16FF353-8441-459E-A50C-B071F53F51B7 for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/DerivedData Exterminator.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2015-03-13 17:59:34.088 xcodebuild[33284:2218432] [MT] PluginLoading: Required plug-in compatibility UUID A16FF353-8441-459E-A50C-B071F53F51B7 for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/ClangFormat.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2015-03-13 17:59:34.088 xcodebuild[33284:2218432] [MT] PluginLoading: Required plug-in compatibility UUID A16FF353-8441-459E-A50C-B071F53F51B7 for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/Alcatraz.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2015-03-13 17:59:34.223 xcodebuild[33284:2218432] [MT] iPhoneSimulator: SimVerifier returned: Error Domain=NSPOSIXErrorDomain Code=53 "Simulator verification failed." UserInfo=0x7fa4054757e0 {NSLocalizedFailureReason=A connection to the simulator verification service could not be established., NSLocalizedRecoverySuggestion=Ensure that Xcode.app is installed on a volume with ownership enabled., NSLocalizedDescription=Simulator verification failed.}
Build settings from command line:
    CODE_SIGN_IDENTITY = 
    CODE_SIGNING_REQUIRED = NO

xcodebuild: error: The directory /Users/jesse/GitHub/JSQDataSourcesKit does not contain an Xcode project.


Failed to generate documentation
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/jazzy-0.1.2/lib/jazzy/sourcekitten.rb:51:in `run_sourcekitten': Running `/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/jazzy-0.1.2/lib/jazzy/sourcekitten/sourcekitten doc` failed:  (RuntimeError)
	from /usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/jazzy-0.1.2/lib/jazzy/doc_builder.rb:55:in `build'
	from /usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/jazzy-0.1.2/bin/jazzy:15:in `<top (required)>'
	from /usr/local/var/rbenv/versions/2.1.5/bin/jazzy:23:in `load'
	from /usr/local/var/rbenv/versions/2.1.5/bin/jazzy:23:in `<main>'
````

It looks like jazzy can't find the Xcode project, which would be in `/Users/jesse/GitHub/JSQDataSourcesKit/JSQDataSourcesKit/` on my machine. It's like it gets confused about the path to the framework project: `JSQDataSourcesKit/JSQDataSourcesKit/ JSQDataSourcesKit.xcodeproj`.

Maybe I'm missing something. Any help would be much appreciated!

**Comment by jpsim** [comment]
I finally got around to looking into this and a fix should be incoming in #189. Thanks for reporting again!

**Comment by jessesquires** [comment]
Hey @jpsim - this is fixed in 0.1.4 :tada: Thanks!

Opening a new, related issue: #196
